bokeh/models/markers.py

```diff
@@ -5,9 +5,20 @@
 # The full license is in the file LICENSE.txt, distributed with this software.
 #-----------------------------------------------------------------------------
 ''' Display a variety of simple scatter marker shapes whose attributes
-can be associated with data columns from ``ColumnDataSources``.
+can be associated with data columns from
+:class:`~bokeh.models.sources.ColumnDataSource` objects.
 
-The full list of markers built into Bokeh is given below:
+.. note::
+    This module is **deprecated since Bokeh 2.3.0**. Please use the
+    :ref:`bokeh.models.glyphs` module instead. Replace all occurrences of
+    ``Marker`` models with :class:`~bokeh.models.glyphs.Scatter` glyphs. For
+    example: instead of ``Asterisk()``, use ``Scatter(marker="asterisk")``.
```

Comment:
New actually want users to only ever user `bokeh.models` directly. Propose the following wording:
> The individual marker classes in this module are **deprecated since Bokeh 2.3.0**. Please use the 
general `Scatter` glyph instead. Replace all occurrences of ``Marker`` models with :class:`~bokeh.models.glyphs.Scatter` glyphs. For
     example: instead of ``Asterisk()``, use ``Scatter(marker="asterisk")``.

Reply:
I see. How about this, to be even shorter:

> The individual marker classes in this module are **deprecated since Bokeh 2.3.0**. Please replace all occurrences of Marker models with :class:~bokeh.models.glyphs.Scatter glyphs. For example: instead of Asterisk(), use Scatter(marker="asterisk").